ui/.eslintrc

```diff
@@ -9,7 +9,9 @@
         "react"
     ],
     "rules":{
-     
+      "no-unused-vars": 1,
+      "no-console": 1,
```

Comment:
I thinks we need to validate that console.log not exists in the code.
WDYT?

Reply:
its forces no-unsed-vars & no-console